index.html

```diff
@@ -18,6 +19,13 @@
         <a href="#">S'inscrire</a>
     </header>
     <main>
+        <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
+        <p>En plein centre ville ou en pleine nature</p>
+        <form action="GET">
```

Comment:
Action cannot be GET, action is url where form is processed so here it's nothing because it's juste a static page so you wan write action="#" and you can can the good attribute method="POST" for example